politeiawww/codetracker/codetracker.go

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2018 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package codetracker
+
+import (
+	cms "github.com/decred/politeia/politeiawww/api/cms/v1"
+)
+
+// CodeTracker interface for getting Code Stats from a git based code tracking
+// site (Github/Gitlab etc).
+type CodeTracker interface {
+	Update(string, string) error                                                  // Use implementation to update the user-information.
```

Comment:
The `Update` function doesn't have any arguments for month/year.  When you call this function, shouldn't you be calling it for a specific time duration?

Reply:
So the Update for the codetracker is just to update the pull requests, reviews and commits for the provided org/repo.  

The start/end provided to the www request applies to updating the users' information for that period.